tsstats/tests/test_ident_map.py

```diff
@@ -24,3 +25,19 @@ def test_ident_map(identmap_clients):
     assert clients['5'] == cl
     assert clients['UID1'] == uidcl
     assert clients['UID5'] == uidcl
+
+
+@pytest.fixture(scope="module")
```

Comment:
Whoops :P I'm a Java programmer at heart and never properly made the switch in my brain.

I'll switch it over to use test function parametrizing as per your other comment.